client/src/pages/Login/useStyles.ts

```diff
@@ -1,8 +1,13 @@
 import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles(() => ({
+  form: {
+    marginTop: '5%',
+  },
+  formBox: {
+    height: '30rem',
```

Comment:
you might not want to use unit size rem for height as it's dependant on font size. pixels would be more suitable